packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -236,6 +242,27 @@ _Props/Attrs_
 -   `markdown` - for retrieving and modifying the markdown value. If the client modifies the markdown value, it will be parsed into a Node using the
     [prosemirror-markdown parser](https://github.com/ProseMirror/prosemirror-markdown/blob/9049cd1ec20540d70352f8a3e8736fb0d1f9ce1b/src/from_markdown.ts#L199).
     The parsed node will then be rendered in the viewer component as rich text.
+-   `external-link` - is a boolean attribute determining whether all absolute links within the viewer component should open in a new tab. By default, all the links will open in the same tab
```

Comment:
The name `external-link` here doesn't seem appropriate. At least in my experience, an "external" link is a link that will take you to a new site. But here, you are using that term to mean a link that will open in a new tab, which is significantly different.

An alternative could be to make this a string attribute called something like `link-target` where the value is one of the valid `target` values on anchor elements (i.e. `"_self" | "_blank" | "_parent" | "_top"`).

Reply:
I have updated the spec as per the suggestion.

Reply:
This API is removed for the scope of this PR and will be updated later.